src/components/CryptoAll_Coins/AllCoins.jsx

```diff
@@ -1,12 +1,68 @@
-import "./AllCoins.css";
+import axios from "axios";
+import { useEffect, useState } from "react";
 import { NavLink } from "react-router-dom";
+import { Symbols } from "recharts";
+
+import "./AllCoins.css";
 
 export function AllCoins() {
+  const [topCoin, setTopCoin] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get(
+        "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=20&page=1&sparkline=false"
+      )
+      .then((response) => {
+        setTopCoin([...response.data]);
+      });
+  }, []);
+
   return (
-    <div className="top_coins_container">
-      <div className="Bitcoin coin_box">BTC/USD</div>
-      <div className="Ethereum coin_box">ETH/USD</div>
-      <div className="DogeCoin coin_box">DEG/USD</div>
+    <div className="top_coin_card">
+      {topCoin.map((coin, i) => {
+        if (i < 3) {
+          return (
+            <>
```

Comment:
make a separate component for all the returned code and call the component here

Reply:
This is also not done

Reply:
sir pls check it is right or not

Reply:
sorry sir i did't push the code now i pushed